blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+
+First, we invited a few colleagues just to try out the mechanics together but pretty soon we saw the magic of the [Metaverse](https://www.gather.town/post/why-the-metaverse-matters) happening.
+
+## Metaverse
+Phillip Wang, the CEO of Gather, has published a little article to the question ["What exactly is the Metaverse?"](https://www.gather.town/post/why-the-metaverse-matters) so I think it's best to cite him:
+
+> Traditionally, physical location has dictated how we live. Most of us have been forced to live near where we work, which means we rarely see friends and family who live elsewhere. For many of those without the means to move, location is a barrier to opportunity altogether.
+> 
+> The Metaverse solves this problem: it's the next iteration of the Internet that brings a sense of place and facilitates rich human connection. In a virtual office you can work alongside your teammates, and strike up casual conversations throughout the day. Together with your friends, you can watch movies in a virtual cinema, enjoy art in a museum, or hang out in a park. You can even go to class with a teacher from halfway across the world, and take a field trip to the pyramids.
+
+This sounds a lot like [Second Life](https://en.wikipedia.org/wiki/Second_Life). Probably the earliest, rather forgotten, type of Metaverse out there. The thing that makes Gather much different is the 2D appearance and the limited interactions. It's not trying to mimic the real world at all. That keeps the focus on the human interactions, which suffered so much during the COVID-19 pandemic when people were forced to move into the home office.
+
+## How we use Gather
+We started using Gather exactly six weeks ago. Your first task is to create your own space. Luckily, they offer plenty of pre-made spaces to pick from. I created a virtual office space, where we could sit at desks or meet at the bar in a kitchen.
+
+![](images/gather/gather-templates.png)
+_Pick an existing gather space from plenty of templates_
+
+From the beginning, I forced myself to keep Gather running every day to act as a role model. Occasionally, I would invite someone from the team to try out Gather with me. 
+
+### All-hands meetings
+Once I reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our office.
+
+![](images/gather/gather-smm.jpg)
+_Our Satellytes folks gathering for our all-hands meeting on a Monday_
+
+The actual meeting is comparable to Zoom or Meet. We broadcast some news and people can share thoughts and ask questions. The time before and after that meeting is where the Metaverse shines ✨. 
```

Comment:
I personally think the existence of broadcasting spots is even one of the more interesting / useful details. 
I wondered in the beginning myself, how the spatial audio thing would work there

Reply:
You are right. Actually we don't even need those because meeting rooms allow to speak to everyone else too. The spotlight is rather to reach everyone in every room. I misused that. I check if I can rewrite this prt.